src/main/java/com/avispl/symphony/dal/infrastructure/management/samsung/smartthings/dto/presentation/DetailViewPresentation.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation.controllableproperties.type.DropdownList;
+import com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation.controllableproperties.type.PushButton;
+import com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation.controllableproperties.type.Slider;
+import com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation.controllableproperties.type.Switch;
+import com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.dto.presentation.controllableproperties.type.TextOrNumber;
+
+/**
+ * Detail View Presentation define how the SmartThings app (or other clients) should present the Attributes and Commands of a Device in detail view/ Aggregated Device view
```

Comment:
For other DTO/constants/enum classes - similar description, that would really give a high-level understanding of its purpose would work great :)

Reply:
I fixed the other DTO like this one